src/_gettsim/sozialversicherung/rente/altersrente/regelaltersrente/altersgrenze.yaml

```
@@ -0,0 +1,106 @@
+---
+regelaltersgrenze:
+  name:
+    de: Einheitliches Eintrittsalter für Regelaltersrente
+    en: Uniform entry age for receiving regelaltersrente - normal retirement age (NRA)
+  description:
+    de: >-
+      § 35 Satz 2 SGB VI
+      Regelaltersgrenze ab der Renteneintritt möglich ist. Wenn früher oder später in
+      Rente gegangen wird, wird der Zugangsfaktor und damit der Rentenanspruch höher
+      oder niedriger, sofern keine Sonderregelungen gelten.
+    en: >-
+      § 35 Satz 2 SGB VI
+      Normal retirement age from which pension can be received. If retirement benefits
+      are claimed earlier or later, the Zugangsfaktor and thus the pension entitlement
+      is higher or lower unless special regulations apply.
+  unit: Years
+  reference_period: null
+  type: scalar
+  1957-02-26:
+    reference: Rentenreformgesetz 1957 BGBl. I S. 88
+    value: 65.0
+  2007-04-19:
+    note: >-
+      Increase of the early retirement age from 65 to 67 for birth cohort 1947-1964. See
+      `regelaltersgrenze_gestaffelt`.
+regelaltersgrenze_gestaffelt:
+  name:
+    de: Gestaffeltes Eintrittsalter für Regelaltersrente nach Geburtsjahr
+    en: Staggered normal retirement age (NRA) for Regelaltersrente by birth year
+  description:
+    de: >-
+      § 35 Satz 2 SGB VI
+      Regelaltersgrenze ab der Renteneintritt möglich ist. Wenn früher oder später in
+      Rente gegangen wird, wird der Zugangsfaktor und damit der Rentenanspruch höher
+      oder niedriger, sofern keine Sonderregelungen gelten.
+    en: >-
+      § 35 Satz 2 SGB VI
+      Normal retirement age from which pension can be received. If retirement benefits
+      are claimed earlier or later, the Zugangsfaktor and thus the pension entitlement
+      is higher or lower unless special regulations apply.
+  unit: Years
+  reference_period: null
+  type: require_converter
+  2007-04-20:
+    reference: RV-Altersgrenzenanpassungsgesetz 20.04.2007. BGBl. I S. 554
+    note: >-
+      Increase of the early retirement age from 65 to 67 for birth cohort 1947-1964.
+      Vertrauensschutz (Art. 56) applies for birth cohorts before 1955 who were in
+      Altersteilzeit before January 1st, 2007 or received "Anpassungsgeld für
+      entlassense Arbeitnehmer des Bergbaus".
+    entry_age_old_regime: 65.0
+    entry_age_new_regime: 67.0
+    max_birthyear_old_regime: 1946
+    min_birthyear_new_regime: 1964
+    1947:
+      base_age: 65
+      months_to_add: 1
```

```suggestion
      months: 1
```

Or any reason to add the extra words? But see general comment below.